posthog/models/feature_flag.py

```diff
@@ -139,3 +185,22 @@ def get_active_feature_flags(team: Team, distinct_id: str) -> List[str]:
         except Exception as err:
             capture_exception(err)
     return flags_enabled
+
+
+# Return a list of with more details about each active flag (including active variant)
```

Comment:
Nit:maybe make this more descriptive - e.g. "Returns a list of feature flags with support for multivariate flags"